src/bins/rmb-peer.rs

```diff
@@ -54,9 +55,9 @@ struct Params {
     #[clap(short, long, default_value_t = String::from("wss://tfchain.grid.tf:443"))]
     substrate: String,
 
-    /// substrate address please make sure the url also include the port number
-    #[clap(long, default_value_t = String::from("wss://relay.grid.tf:443"))]
-    relay: String,
+    /// set of relay Urls (max 3) please ensure url contain a domain
+    #[clap(long, num_args = 1..=3 , default_values = ["wss://relay.grid.tf:443"])]
```

Comment:
It can’t be infinite. The tide to the storage item for the relay field on chain which is limited by a bound on its size. Currently, the size on chain is limited to 1024 which theoretically allows for up to 4 hosts domain name based on the standards. However, I relaxed it a bit and allowed only 3 but it can be up to 4. I’m sticking with this unless further planning is done from tfchain side.”